Formula/fargatecli.rb

```diff
@@ -0,0 +1,25 @@
+class Fargatecli < Formula
+  desc "CLI for AWS Fargate"
+  homepage "https://github.com/awslabs/fargatecli"
+  url "https://github.com/awslabs/fargatecli/archive/v0.3.1.tar.gz"
+  sha256 "fd8627e6ab85d70f6b2633d8b35cfb1daad232ed0a29c72df57b9869810ab221"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args
+    prefix.install_metafiles
+  end
+
+  def caveats
+    <<~EOS
+      Before you can use this tool you should export some variables to your $SHELL.
+        export AWS_ACCESS_KEY="<Your AWS Access ID>"
+        export AWS_SECRET_KEY="<Your AWS Secret Key>"
+    EOS
+  end
+
+  test do
+    assert_match "Deploy serverless containers", shell_output("#{bin}/fargatecli help")
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Ok, no problem.... just for your information:

```
homebrew-core/Formula] master* » rg -c -e asset_match  -e help |wc -l
     351
```

> If the software cannot function without  credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.

I am going to assume that we can make some requests to AWS for every  pipeline testing this... I hope it is ok. 
This is a go binary, the build coverage is coming from the awslabs repository.... anyway... i'm gonna work on your point.